code/game/objects/structures/shower.dm

```diff
@@ -164,10 +164,7 @@
 
 		if(ishuman(M))
 			var/mob/living/carbon/human/H = M
-			if(check_clothes(L))
-				to_chat(L, "<span class='warning'>You shower with your clothes on, and feel like an idiot.</span>")
-				SEND_SIGNAL(L, COMSIG_ADD_MOOD_EVENT, "badshower", /datum/mood_event/idiot_shower)
-			else
+			if(!check_clothes(L))
```

Comment:
Now people cant tell why showering isnt cleaning them; could you add a message or something for if it doesnt work?

Reply:
yeah this really, really feels like a writing/error message problem more than a mechanical problem

but this is from the people who brought us "showering is fetish content" so

Reply:
Yeah, I can add a message. One moment.

Reply:
Right, so due to how the shower code works, the user gets spammed with this message. As such, it only appears when their Hygiene level is 75 or less, which is usually the only time they'd care about it anyway.

Reply:
you could add a cooldown to it